### PR TITLE
Update dashboard FAQ

### PR DESCRIPTION
**Pull request by shhdgit**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Added a new FAQ for TiDB Dashboard

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- Other reference link(s): [tidb-dashboard#902](https://github.com/pingcap/tidb-dashboard/issues/902)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shhdgit** [comment]
related to #5522

File: dashboard/dashboard-faq.md [hunk 1]
```diff
@@ -58,3 +58,17 @@ If your deployment tool is TiUP, take the following steps to solve this problem.
...
 
 The possible reason is that you have enabled the `prepared-plan-cache` feature of TiDB. As an experimental feature, when enabled, `prepared-plan-cache` might not function properly in specific TiDB versions, which could cause this problem in TiDB Dashboard (and other applications). You can disable `prepared-plan-cache` by updating [TiDB Configuration file](/tidb-configuration-file.md#prepared-plan-cache) to solve this problem.
+
+### An `unknown field` error is shown in **Slow Queries** page
+
+If the `unknown field` error appears on the Slow Queries page after the cluster upgrade, that is a compatibility issue caused by the difference between TiDB Dashboard server fields (which may be updated) and user preferences fields (which are in the browser cache). This issue has been fixed in v4.0.14/v5.0.3 and later, older versions can be resolved by clearing the browser Local Storage.
+
+1. Open TiDB Dashboard page
+
+2. Click **Customize and control Google Chrome** and then select **More Tools > Developer Tools**.
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:68]: Users might not necessarily use Chrome.

Comment (shhdgit) [reply dashboard/dashboard-faq.md:68]: Yeah, right. Thanks!

File: dashboard/dashboard-faq.md [hunk 2]
```diff
@@ -58,3 +58,17 @@ If your deployment tool is TiUP, take the following steps to solve this problem.
 ### An `invalid connection` error is shown in **Top SQL Statements** and **Recent Slow Queries** on the Overview page
 
 The possible reason is that you have enabled the `prepared-plan-cache` feature of TiDB. As an experimental feature, when enabled, `prepared-plan-cache` might not function properly in specific TiDB versions, which could cause this problem in TiDB Dashboard (and other applications). You can disable `prepared-plan-cache` by updating [TiDB Configuration file](/tidb-configuration-file.md#prepared-plan-cache) to solve this problem.
+
+### An `unknown field` error is shown in **Slow Queries** page
+
+If the `unknown field` error appears on the Slow Queries page after the cluster upgrade, that is a compatibility issue caused by the difference between TiDB Dashboard server fields (which may be updated) and user preferences fields (which are in the browser cache). This issue has been fixed in v4.0.14/v5.0.3 and later, older versions can be resolved by clearing the browser Local Storage.
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:64]: For dev users, this issue has already been fixed. I think "If you are using v4.0.14/v5.0.3 or an earlier v4.0/v5.0 version, you can resolve this issue by clearing your browser cache" is not necessary and can be removed in dev version.
```suggestion
If the `unknown field` error appears on the **Slow Queries** page after the cluster upgrade, the error is related to a compatibility issue caused by the difference between TiDB Dashboard server fields (which might be updated) and user preferences fields (which are in the browser cache). This issue has been fixed. If you are using v4.0.14/v5.0.3 or an earlier v4.0/v5.0 version, you can resolve this issue by clearing your browser cache.
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: dashboard/dashboard-faq.md [hunk 3]
```diff
@@ -58,3 +58,25 @@ If your deployment tool is TiUP, take the following steps to solve this problem.
...
+
+To clear your browser cache, take the following steps:
+
+1. Open TiDB Dashboard page.
+
+2. Open Developer Tools. Different browsers have different ways of opening Developer Tools. After clicking the **Menu Bar**:
+
+  - Firefox: Menu ➤ Web Developer ➤ Toggle Tools, or Tools ➤ Web Developer ➤ Toggle Tools.
+  - Chrome: More tools ➤ Developer tools.
+  - Safari: Develop ➤ Show Web Inspector. If you can't see the Develop menu, go to Safari ➤ Preferences ➤ Advanced, and check the Show Develop menu in menu bar checkbox.
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:74]:
```suggestion
    - Firefox: Menu ➤ Web Developer ➤ Toggle Tools, or Tools ➤ Web Developer ➤ Toggle Tools.
    - Chrome: More tools ➤ Developer tools.
    - Safari: Develop ➤ Show Web Inspector. If you can't see the Develop menu, go to Safari ➤ Preferences ➤ Advanced, and check the Show Develop menu in menu bar checkbox. 
```

File: dashboard/dashboard-faq.md [hunk 4]
```diff
@@ -58,3 +58,25 @@ If your deployment tool is TiUP, take the following steps to solve this problem.
 ### An `invalid connection` error is shown in **Top SQL Statements** and **Recent Slow Queries** on the Overview page
 
 The possible reason is that you have enabled the `prepared-plan-cache` feature of TiDB. As an experimental feature, when enabled, `prepared-plan-cache` might not function properly in specific TiDB versions, which could cause this problem in TiDB Dashboard (and other applications). You can disable `prepared-plan-cache` by updating [TiDB Configuration file](/tidb-configuration-file.md#prepared-plan-cache) to solve this problem.
+
+### An `unknown field` error is shown in **Slow Queries** page
+
+If the `unknown field` error appears on the **Slow Queries** page after the cluster upgrade, the error is related to a compatibility issue caused by the difference between TiDB Dashboard server fields (which might be updated) and user preferences fields (which are in the browser cache). This issue has been fixed. If you are using v4.0.14/v5.0.3 or an earlier v4.0/v5.0 version, you can resolve this issue by clearing your browser cache.
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:64]:
```suggestion
If the `unknown field` error appears on the **Slow Queries** page after the cluster upgrade, the error is related to a compatibility issue caused by the difference between TiDB Dashboard server fields (which might be updated) and user preferences fields (which are in the browser cache). This issue has been fixed.
```

File: dashboard/dashboard-faq.md [hunk 5]
```diff
@@ -58,3 +58,25 @@ If your deployment tool is TiUP, take the following steps to solve this problem.
 ### An `invalid connection` error is shown in **Top SQL Statements** and **Recent Slow Queries** on the Overview page
 
 The possible reason is that you have enabled the `prepared-plan-cache` feature of TiDB. As an experimental feature, when enabled, `prepared-plan-cache` might not function properly in specific TiDB versions, which could cause this problem in TiDB Dashboard (and other applications). You can disable `prepared-plan-cache` by updating [TiDB Configuration file](/tidb-configuration-file.md#prepared-plan-cache) to solve this problem.
+
+### An `unknown field` error is shown in **Slow Queries** page
+
+If the `unknown field` error appears on the **Slow Queries** page after the cluster upgrade, the error is related to a compatibility issue caused by the difference between TiDB Dashboard server fields (which might be updated) and user preferences fields (which are in the browser cache). This issue has been fixed.
```
Comment (TomShawn) [inline dashboard/dashboard-faq.md:64]:
```suggestion
If the `unknown field` error appears on the **Slow Queries** page after the cluster upgrade, the error is related to a compatibility issue caused by the difference between TiDB Dashboard server fields (which might be updated) and user preferences fields (which are in the browser cache). This issue has been fixed. If your cluster is earlier than v5.0.3 or v4.0.14, perform the following steps to resolve the issue:
```

**Pull request merged**